guides/source/active_model_basics.md

```diff
@@ -388,25 +1110,36 @@ class Person
   attr_accessor :name
 
   def attributes
-    { 'name' => nil }
+    { "name" => nil }
   end
 end
 ```
 
-The `as_json` method, similar to `serializable_hash`, provides a Hash representing
-the model.
+The `as_json` method, similar to `serializable_hash`, provides a Hash
```

Comment:
Hash can be lowercased.

Reply:
String can be lowercased.

Reply:
I could use clarity on when to use hash vs Hash - I guess `Hash` would probably be when we're referring to the actual [Class method](https://ruby-doc.org/core-2.5.1/Hash.html) rather than just talking about the data structure? I noticed that they're being capitalized in https://guides.rubyonrails.org/getting_started.html which confused me slightly.